cloud-gateway-service/build.gradle

```diff
@@ -78,6 +79,8 @@ dependencies {
 
     implementation libs.swagger2.parser
     implementation libs.swagger3.parser
+    implementation libs.thymeleaf
```

Comment:
Do we need those libs once the starter is there?

Reply:
I'm not sure whether this is part of the starter. I will check

Reply:
yes, good question, I will test using only the starter one